templates/extras/calc.md

```diff
@@ -52,31 +30,20 @@ proof is complete. Use `_` as in the example above or `sorry` to stand
 for missing justifications. `sorry` will supress error messages
 entirely, while `_` will generate a guiding error message.
 
-If the structure of calc is incorrect (e.g., missing `:` or the
+If the structure of calc is incorrect (e.g., missing `:=` or the
 justification after it), you may see error messages that are obscure
-and/or red squiggles that end up under a random `...`. To avoid these,
+and/or red squiggles that end up under a random `_`. To avoid these,
 you might first populate a skeleton proof such as:
 
 ```lean
 example (A B C D : ℝ ) : A = D :=
-calc A = B : sorry
-...    = C : _
-...    = D : sorry
+calc A = B := sorry
+     _ = C := _
+     _ = D := sorry
 ```
 
 and then fill in the `sorry` and `_` gradually.
 
-In tactic mode calc should be terminated with a comma:
-```lean
-have H : A = D,
-{ calc A = B : sorry
-  ...    = C : sorry
-  ...    = D : _,
-  sorry },
-```
-and the `_` can be left in as they generate a subgoal to be resolved
-after calc (here by the last `sorry`).
-
 (Idle thought: could one write a VS Code snippet to write this skeleton?)
```

Comment:
This comment is outdated. There is now a code action doing it. More generally this documentation should now explain the calc widget.

Reply:
Do you have details of the calc widget? I can't work it out

Reply:
Didn't know about this having a look into this

Reply:
Sorry can't work out how to use this

Reply:
Apparently you've made a video about this I'll go find it

Reply:
added a link to a video of you demonstrating this.